docs/advanced.md

```diff
@@ -44,6 +45,48 @@ which can be used to show custom page numbers or add other elements to the UI.
 - `step`: (number) the current visible step in an Appear or Step component
 - Each slide includes a `meta` object with a `notes` field when the Notes component is used within a slide
 
+#### Example
+
+The example below will display the current slide out of the total amount of slides.
+
+```js
+import React, { useEffect, useRef, useState } from 'react'
+import { css } from '@emotion/core'
+
+function AtTheBottomCenter ({ children }) {
+  const ref = useRef();
+  const [left, setLeft] = useState(0)
+
+  useEffect(function WindowSize () {
+    const bodyWidth = document.body.clientWidth
+    const width = ref.current.clientWidth
+
+    setLeft((bodyWidth / 2) - (width / 2))
+  })
+
+  const css = css`
+    position: absolute;
+    bottom: 0;
+    color: #ffffff;
+  `
```

Comment:
Thought you could do multiline suggested changes, but I can't figure it out... I'd prefer this inline in the component below, but the CSS could be:

```js
const css = {
  position: 'absolute',
  bottom: 0,
  left: '50%',
  transform: 'translateX(-50%)',
}
```

Or, it could span the entire width and center the text:
```js
const css = {
  position: 'absolute',
  left: 0,
  right: 0,
  bottom: 0,
  textAlign: 'center',
}
```

Reply:
Yes I agree, using css here would be a lot better.